### PR TITLE
Release 2.1.1

### PR DESCRIPTION
**Pull request by d4straub**
Current version 2.1.0 doesnt work with the newest nextflow version 21.10.x, so that minor release is needed.
Changes are added Prokka (prokaryotic gene finding & annotation) per MAG, prodigal (prokaryotic gene finding) on metagenome and several bug fixes.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by ggabernet — APPROVED** [review]
Looks good to me!

File: CHANGELOG.md
Comment (jfy133) [inline CHANGELOG.md:6]:
```suggestion
## v2.1.1 - 2021/11/25
```

**Review by jfy133 — APPROVED** [review]
LGTM :+1:

File: CHANGELOG.md
Comment (d4straub) [reply CHANGELOG.md:6]: Thanks! Indeed, I was a little optimistic with the date.

**Comment by d4straub** [comment]
Thanks for the reviews! I am running the large test run on our cluster right now, will wait until thats finished before I merge.

**Comment by jfy133** [comment]
@d4straub beforeyou go, in a separte test I noticed the following typo:

https://github.com/nf-core/mag/blob/0b51bd10aa5277e46b6f5ac6b1f01a86a8f6b089/lib/WorkflowMag.groovy#L83-L119

specififed -> specified

**Comment by d4straub** [comment]
Good catch! Seems like a copy paste error, I replaced all occurrences of `specififed` with `specified` in https://github.com/nf-core/mag/pull/262

**Comment by jfy133** [comment]
Approved

**Pull request merged**
